### PR TITLE
Automate configuration of drivers

### PR DESCRIPTION
**Pull request by AlanGriffiths**
@RAOF you may have a better idea of how to configure the graphics stack, but this ought to work. (You'll need to add a 'mir-quirks' file to the MKT snap.)

### CODE REVIEW

File: snap/hooks/configure [hunk 1]
```diff
@@ -62,6 +62,12 @@ vt=0
 EOT
 fi
 
+
+if [ -e "$SNAP/graphics/lib/mir-quirks" ]
```
Comment (RAOF) [inline snap/hooks/configure]: I'd probably put this in `graphics/etc/mir-quirks`? I don't know if there's much value in that other than personal “configuration stuff goes in `etc`” vibes.

Comment (RAOF) [inline snap/hooks/configure]: We might want this to use the `-f` test? We don't want to accidentally pick up a `mir-quirks` directory.

**Review by RAOF — APPROVED** [review]
I am in favour of this over #12. Do the minor stylistic changes, or not.

bors delegate=AlanGriffiths

**Review by RAOF — CHANGES_REQUESTED** [review]
bors cancel

Actually, shouldn't this be an [`interface`](https://snapcraft.io/docs/interface-hooks) hook? By my reading the [`configure` hook](https://snapcraft.io/docs/supported-snap-hooks#heading--the-configure-hook) is only called on initial install, refresh, or `snap set`, whereas we want to re-check this whenever the `graphics-core20` content interface is connected.

File: snap/hooks/configure (hunk 1 above)
Comment (AlanGriffiths) [reply snap/hooks/configure]: Oops! I intended to just drop the "lib". But "etc" is good.

**Comment by AlanGriffiths** [comment]
@RAOF I think that covers all your (perfectly valid) concerns.

**Comment by AlanGriffiths** [comment]
I'm going to merge this. What's the worst that could happen?

bors r=@RAOF

**Pull request merged**